SCClassLibrary/Common/Control/Server.sc

```diff
@@ -369,9 +369,12 @@ Server {
 	var <pid, serverInterface;
 	var pidReleaseCondition;
 
+	var serverNodeTree;
+
 	*initClass {
 		Class.initClassTree(ServerOptions);
 		Class.initClassTree(NotificationCenter);
+		Class.initClassTree(ServerNodeTree);
```

Comment:
since there is no `initClass` in `ServerNodeTree`, this is not necessary.

Reply:
Fixed.